### PR TITLE
Renders DisplayColours with AI provided colours after pressing `get reccomendations` button

### PR DESCRIPTION
**Pull request by frosty8104**
Filtering colour request from Groq to display colour scheme on UI (I think?)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -60,6 +63,32 @@ function App() {
       return { ...currForm, [changedField]: newValue };
     });
   };
+  const colors = [
```
Comment (mnichols08) [inline src/App.jsx]: this is now an unused variable, we can delete

File: src/components/UserData.jsx
Comment (mnichols08) [inline src/components/UserData.jsx:1]: I think this file was deleted by a previous PR, it can also be removed before we merge this

**Review by mnichols08 — CHANGES_REQUESTED** [review]
I am going to request these couple of changes to be made before merging. 

Great job with implementing, it appears that this is working property.

Minor changes requested:
- [x] Remove the `const colors = [...]` declaration as it is an unused variable.
- [x] Remove the `UserData.jsx` file as it was deleted in a previous PR.

**Review by mnichols08 — APPROVED** [review]
Looking good to me =)

File: src/components/DisplayColours.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    <h3>Thank you for choosing Color Compass</h3>
+    {colors.map(color => (
+        <div key={color.hex}>
+            <div className={`bg-${color.hex.slice(1)}-300 rounded-full p-6 inline-block`}
+            style={{ backgroundColor: color.hex }}></div>
+        <div className={`bg-${color.hex.slice(1)}-300 rounded-full inline-block ml-4 p-4`}>
+        <p>Hex: {color.hex}z</p>
+        <p>Name: {color.name}</p>
+    <div>
+        <p>Description: {color.description}</p>
```
Comment (KseniiaRiabova) [inline src/components/DisplayColours.jsx]: Line 9: Tailwind CSS doesn't support dynamic class names:
{`bg-${color.hex.slice(1)} this doesn't work properly, style is working.
Line 11: why do we need it?
Line:12/13/15 are made not according to Figma design

**Review by KseniiaRiabova — CHANGES_REQUESTED** [review]

File: src/components/DisplayColours.jsx (hunk 2 above)
Comment (frosty8104) [reply src/components/DisplayColours.jsx]: Ok, any suggestions on how to change those last lines so they are according to Figma design? I took out the color.hex.slice part so will push those changes up, also deleted line 11

File: src/components/DisplayColours.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    <>
+    <h3>Thank you for choosing Color Compass</h3>
+    {colors.map(color => (
+        <div key={color.hex}>
+            <div className={`rounded-full p-6 inline-block`}
+            style={{ backgroundColor: color.hex }}></div>
+        <p>Hex: {color.hex}</p>
+        <p>Name: {color.name}</p>
+    <div>
+        <p>Description: {color.description}</p>
```
Comment (KseniiaRiabova) [inline src/components/DisplayColours.jsx:14]: These lines need to be just {color.hex} / {color.name} / {color.description} ( not Hex:{color.hex} ) according to Figma design

**Review by KseniiaRiabova — APPROVED** [review]

**Review by Valeriusdev — APPROVED** [review]

**Pull request merged**